src/Stratis.Bitcoin/P2P/Peer/NetworkPeerServer.cs

```diff
@@ -208,5 +232,28 @@ private NetworkPeerConnectionParameters CreateNetworkPeerConnectionParameters()
             param2.AddressFrom = myExternal;
             return param2;
         }
+
+        /// <summary>
+        /// Determine if an inbound TCP client (Node) can be closed.
```

Comment:
Seems to be a mismatch in the language here and in the returns.....   if it can be closed or  it *must* definitely be closed ?

Reply:
And if it is the case that it must be closed, then perhaps this method is responsible for the closing too rather than returning the answer to the query for the client call to decide?

Reply:
Re first comment, I changed the code, but forgot to change the comment..,. will update, thanks.

Reply:
Re second point, I was closing it in the function then moved it out.  Now you've mentioned this I agree it makes more sense, i'll update, thanks.